### DISCUSSION TITLE
Multiplus-2 GX compatible?

### DISCUSSION THREAD

**Discussion by greymda** [freakent/dbus-mqtt-devices#135] [Q&A]
Hi and thanks for your work!
Is this project compatible with a Multiplus-2 with built-in GX?

**Comment by freakent** [comment]
Someone recently posted on here that they were using this driver with Multiplus II GX. I don't have access to one so I can't personally verify it. All GX devices will automatically reboot if they start to get low on memory  so if you get random reboots it might be too much for it. 

What are you planning to integrate with the driver?

**Comment by BobvanderWeg** [comment]
I'm using this driver on a Multiplus-2 GX. 
Using 2 temperature and the EV-charger via this driver.
